docs/tfuser/readme.md

```diff
@@ -15,113 +15,175 @@ It has 3 main commands:
 Here is a full example how you would use tfuser to create a network schema.
 
 1. create a network from scratch
-`tfuser generate --schema network.json network create --node {nodeID}`
+`tfuser generate network create --name {NETWORK_NAME} --cidr ${CIDR}`
 
-the `--node` flag point to the node ID of the exit node we have chosen for the network.  
-the value of the `--schema` flag is a path where the provisoin schema will be written by tfuser. If not specified the schema will be printed to stdout.
+`NETWORK_NAME`: Arbitrary name of your network.
+`CIDR`: The prefix which will be used by the network resources in the network.
 
-2. add another node to the network:
-`tfuser generate --schema network.json network add-node --node {nodeID}`
+The schema will be printed on stdout, and should be saved for further actions.
```

Comment:
We should also introduce the --schema flag here since it is then re-used later on in the doc.